tests/std/tests/P0475R1_P0591R4_uses_allocator_construction/test.cpp

```diff
@@ -173,6 +174,11 @@ constexpr bool test_P0591R4() {
     return true;
 }
 
+void test_GH_2021() { // compile-only

```

Comment:
Our `// COMPILE-ONLY` comments are conventionally VERY SHOUTY.
```suggestion
void test_GH_2021() { // COMPILE-ONLY
```